actors/cloudbroker/cloudbroker__qos/methodclass/cloudbroker_qos.py

```diff
@@ -72,3 +74,58 @@ def limitInternetBandwith(self, cloudspaceId, rate, burst, **kwargs):
         vfw = self.vcl.virtualfirewall.get(vfwid)
         self.acl.executeJumpscript('cloudscalers', 'limitpublicnet', gid=vfw.gid, nid=vfw.nid,
                                    args={'networkId': cloudspace.networkId, 'rate': rate, 'burst': burst})
+
+    def vm_events(self, event, state, name):
+        """
+        This will handle vm qos events and send the relevant users emails notifying them
+
+        param:event the event to handle
+        param:state the state of the event (WARNING, SUCCESS)
+        param:name the name of the vm
+        """
+        vms = self.cb.cbcl.VMachine.objects(id=name.replace('vm-', ''))
```

Comment:
use the .first() method of mongoegnine it will return the first object or None incaise its an empty list